.travis.yml

```diff
@@ -14,7 +14,7 @@ env:
     - RAILS='~> 4.2.0'
     - RAILS='~> 5.0.0'
     - RAILS='~> 5.1.0'
-    - RAILS='~> 5.2.0.beta2'
+    - RAILS='~> 5.2.0.rc2'
```

Comment:
@bbonislawski 

Don't you need to update `env: RAILS='~> 5.2.0.beta2'` line in `allowed_failures` section?

Reply:
Nice catch @eitoball